lib/unobtainium/drivers/appium.rb

```diff
@@ -147,6 +147,10 @@ def resolve_options(label, options)
                                                    .recursive_merge(options[:caps])
             options.delete(:desired_capabilities)
             options.delete('desired_capabilities')
+          else
+            options["desired_capabilities"] = options["desired_capabilities"].merge({javascript_enabled: true})
```

Comment:
You can omit the curly braces in the `merge` method argument and it is also the preferred style, in general.